packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -423,6 +428,34 @@ describe("deploy", () => {
 		`);
 		});
 
+		it("should warn user when worker has deployment with multiple versions", async () => {
```

Comment:
Can we have a test that checks what happens if the user says yes? 
Also what happens if the console is non-interactive? Can that be tested too?